README.md

```diff
@@ -1,23 +1,27 @@
 Formality
 =========
 
-Formality is a proof and programming language, as well as a standard format for
-algorithms and proofs. It exists to fill a hole in the current market: there
-aren't many programming languages featuring theorem proving that are 1.
-portable, 2. efficient, 3. accessible.
-
-**Portability** makes it available everywhere, like a JSON of code. We
-accomplish that by bootstrapping: we design a minimal core that can be quickly
-implemented anywhere, and build the rest of the language on top of it.
-
-**Efficiency** is required if we want the language to be used for production
-software. We accomplish that by compiling Formality to a massively parallel,
+Formality is a dependently-typed programming language that enables developers to
+use formal proofs to make their applications safer, faster and friendlier.
```

Comment:
The intention of this paragraph was to convey 3 things. 

1. That Formality is a proof language that is, unlike Agda/Coq, designed to be practical, efficient, easy.

2. That it is a lightweight proof-interchange format, like JSON is a data-interchange format.

3. That it can be used as a programming language and that you'll like it if you do so, even if you are a regular developer.

Trying to convey those 3 things in the least amount of words is challenging. I don't think neither mine nor your sentence does that really well, but I prefer mine's.

Reply:
Okay, I will rewrite it to better convey this